frontend/jupyter/src/app/pages/form/form-new/form-image-custom/form-image-custom.component.ts

```diff
@@ -0,0 +1,210 @@
+import {
+  Component,
+  OnInit,
+  Input,
+  OnDestroy,
+  LOCALE_ID,
+  Inject,
+} from '@angular/core';
+import {
+  FormGroup,
+  Validators,
+  ValidatorFn,
+  AbstractControl,
+} from '@angular/forms';
+import { Subscription } from 'rxjs';
+import { environment } from '@app/environment';
+import { DomSanitizer } from '@angular/platform-browser';
+import { MatIconRegistry } from '@angular/material/icon';
+import { V1Namespace } from '@kubernetes/client-node';
+import { Config } from 'src/app/types';
+import { MatCheckboxChange } from '@angular/material/checkbox';
+
+@Component({
+  selector: 'app-form-image-custom',
+  templateUrl: './form-image-custom.component.html',
+  styleUrls: ['./form-image-custom.component.scss'],
+})
+export class FormImageCustomComponent implements OnInit, OnDestroy {
```

Comment:
I think you should re-evaluate what needs to be in this component. From what I can tell, this was created from a copy paste of the form-image component, and there's a lot of lingering code from that component that is not needed in here.